R/LSBuffonsneedlemanipulation.R

```diff
@@ -0,0 +1,197 @@
+  summaryTable <- createJaspTable(title = gettext("Summary Table"))
+  summaryTable$position <- 1
+  summaryTable$dependOn(c("k", "n", "a", "b", "length", "CI"))
+  #summaryTable$addCitation("JASP Team (2018). JASP (Version 0.9.2) [Computer software].")
+  
+  summaryTable$addColumnInfo(name = "NumCrosses", title = gettext("Crosses"), type = "integer")
+  summaryTable$addColumnInfo(name = "NumObservations", title = gettext("Observations"), type = "integer")
+  summaryTable$addColumnInfo(name = "Median", title = gettextf("Median for %s", "\u03c0"),   type = "number")
+  summaryTable$addColumnInfo(name = "lowerCI", title = gettext("Lower"), type = "number", 
+                            overtitle = gettextf("%s%% Credible Interval", options[["CI"]]*100))
+  summaryTable$addColumnInfo(name = "upperCI", title = gettext("Upper"), type = "number", 
+                            overtitle = gettextf("%s%% Credible Interval", options[["CI"]]*100)) 
+
+
+  # fill in the table
+  CI95lower <- 2 * l / (qbeta((1-options[["CI"]])/2, options[["k"]], options[["n"]] - options[["k"]], lower.tail = FALSE) * d)
+  CI95lower = round(CI95lower, digit = 2)
+  
+  med <- 2 * l / (qbeta(.5, options[["k"]], options[["n"]] - options[["k"]], lower.tail = FALSE) * d)
+  med = round(med, digit = 2)
+  
+  CI95upper <- 2 * l / (qbeta(1-(1-options[["CI"]])/2, options[["k"]], options[["n"]] - options[["k"]], lower.tail = FALSE) * d)
+  CI95upper = round(CI95upper, digit = 2)
+  
+  summaryTable$addRows(list(NumCrosses = options[["k"]], NumObservations = options[["n"]],
+                           lowerCI = CI95lower, Median = med,   upperCI = CI95upper))
+  jaspResults[["summaryTable"]] <- summaryTable
+}
+     
+
+.buffonsNeedleManipulationPropDistPlot <- function(jaspResults, options){
+  if(!is.null(jaspResults[["propDistPlot"]])) return()
+  d <- 5
+  l <- options[["length"]]*d/100
+  ## 1. prior and posterior plot for proportion of crosses
+  if (options[["showPropDistPlot"]]){
+    propDistPlot <- createJaspPlot(title = gettext("Prior and Posterior for Proportion of Crosses"),
+                                   width = 480, height = 320)
+    propDistPlot$position <- 2
+    #propDistPlot$dependOn(c("k", "n", "a", "b", "length", "CI", "showPropDistPlot"))
+    
+    propDistPlot$dependOn(optionsFromObject = jaspResults[["summaryTable"]], 
+                          options = c("showPropDistPlot", "legendPropDistPlot"))
+    #propDistPlot$addCitation("JASP Team (2018). JASP (Version 0.9.2) [Computer software].")
+    
+    # values
+    
+    xValue <- seq(0,1,0.005)
+    propPost <- dbeta(xValue, options[["a"]] + options[["k"]], options[["b"]] + options[["n"]] - options[["k"]])
+    propPrior <-dbeta(xValue, options[["a"]], options[["b"]])
+    dataProp = data.frame(values = c(xValue, xValue),
+                          density = c(propPost, propPrior),
+                          group = c(rep(gettext("Posterior"),201), rep(gettext("Prior"),201))
+    )
```

Comment:
Because you later on use the `labels` to get the translated strings, you don't need to `gettext` the values here

```suggestion
    dataProp = data.frame(values = c(xValue, xValue),
                          density = c(propPost, propPrior),
                          group = c(rep("Posterior",201), rep("Prior",201))
    )
```